src/app/about/page.tsx

```diff
@@ -1,47 +1,56 @@
-import React from 'react';
+'use client';
 
-import AboutText from '@/components/About/AboutText';
-import HungerMapLiveSuperscript from '@/components/About/HungerMapLiveSuperscript';
-import StyledLink from '@/components/About/StyledLink';
-import AccordionContainer from '@/components/Accordions/AccordionContainer';
+import React, { Suspense, useState } from 'react';
+
+import LiveSuperscript from '@/components/About/LiveSuperscript';
+import DocsSearchBar from '@/components/Search/DocsSearchBar';
+import SearchableSection from '@/components/Search/SearchableSection';
+import { aboutTextElements } from '@/domain/constant/about/aboutTextElements';
 import generalFaqItems from '@/domain/constant/about/generalFaqItems';
-import predictionFaqItems from '@/domain/constant/about/predictionFaqItems';
+import predictionFaqItems, { predictionFaqText } from '@/domain/constant/about/predictionFaqItems';
 import realTimeFaqItems from '@/domain/constant/about/realTimeFaqItems';
 
 function Page() {
+  const SECTIONS = [
+    { textElements: aboutTextElements },
+    { heading: 'General Questions', accordionItems: generalFaqItems },
+    { heading: 'Near real-time food security continuous monitoring', accordionItems: realTimeFaqItems },
+    { heading: 'Predictive analysis', textElements: predictionFaqText, accordionItems: predictionFaqItems },
+  ];
+
+  const [searchWords, setSearchWords] = useState<string[]>([]);
+  const [visibleAccordions, setVisibleAccordions] = useState<Set<string>>(new Set());
+
   return (
-    <>
-      <section>
-        <h1>
-          About <HungerMapLiveSuperscript />
+    <Suspense>
```

Comment:
Add PageSkeleton to the Suspense, currently nothing is shown, not even indicating that we are loading data. Please add Skeleton for the searchBar, some text paragraphs and accordions. Maybe take inspiration from Linkedin or Notion Loading indicators there